features/storage/csi_clone.feature

```diff
@@ -0,0 +1,58 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use "oc get sc" can find all deployed stroage class.
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
+  Scenario: Clone a pvc on OSP in the same namespace is successfully
+    # Step 1
+    Given the master version >= "4.7"
+    Given I have a project
+    Given I use the "<%= project.name %>" project
+    When I obtain test data file "storage/csi/pod_pvc_sc.yaml"
+    And I run the :create client command with:
+      | f | pod_pvc_sc.yaml |
+    Then the step should succeed
+    And the pod named "mypod" becomes ready
+    And the "mypvc" PVC becomes :bound
+    # Step 2
+    Given 1 pod becomes ready with labels:
+      | name=frontendhttp |
+    # Because the execution is too fast, we have to split test file creation and writing text into 2 commands: 
+    # touch and echo, and use the wait step.
+    And I wait for the steps to pass:
+    """
+    When I execute on the pod:
+      | sh | -c | touch /mnt/local/testfile |
+    Then the step should succeed
+    """
+    And I wait for the steps to pass:
+    """
+    When I execute on the pod:
+      | sh | -c | echo "clone test" > /mnt/local/testfile |
+    Then the step should succeed
+    """
+    And I wait for the steps to pass:
+    """
+    When I execute on the pod:
+      | sh | -c | more /mnt/local/testfile |
+    Then the output should contain "clone test"
+    """
+    # Step 3
+    Given I obtain test data file "storage/csi/pod_clone.yaml"
```

Comment:
You are having Pod and PVC specs together in one file, you need to name the file to reflect that. The name `pod_clone.yaml` looks like a yaml for Pod, when it also specifies a PVC.

Reply:
@jhou1 the file name is same as it in bug, [https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-27615](url)

Reply:
The file name and structure are totally against the test case.

Reply:
OK, don't worry about that test case in polarion, let's make sure our automation files and scripts have clean and consistent naming.